test/ViewModel/Factory/SiteHeaderFactoryTest.php

```diff
@@ -39,14 +41,17 @@ public function it_links_directly_to_the_submit_site_when_the_feature_flag_is_di
     {
         $siteHeader = $this->siteHeaderFactory->createSiteHeader();
 
-        $this->assertSame('http://submit.elifesciences.org/path', $siteHeader['secondaryLinks']['linkedItems'][2]['button']['path']);
+        $this->assertSame(
+            'http://submit.elifesciences.org/path',
+            $siteHeader['secondaryLinks']['linkedItems'][2]['button']['path']
+        );
     }
 
     /**
      * @test
      * @backupGlobals enabled
      */
-    public function it_links_to_the_submit_route_when_the_feature_flag_is_enable()
+    public function it_links_directly_to_the_submit_site_when_the_feature_flag_is_enabled_and_you_are_not_logged_in()
```

Comment:
as far as I can tell this will result in this page being shown:
https://staging--xpub.elifesciences.org/login
This doesn't force a login, as the users can go to the author guide or to other public pages linked in the top navigation bar.